src/executables/util_cs.c

```diff
@@ -3156,6 +3158,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
   LOG_LSA master_eof_lsa, applied_final_lsa;
   LOG_LSA copied_append_lsa, copied_eof_lsa;
   LOG_LSA initial_copied_append_lsa, initial_applied_final_lsa;
+  char *ha_node_list_p = NULL;
+  char **ha_node_list_pp = NULL;
+  char *start_node;
```

Comment:
doesn't need to be initialized as NULL ?

Reply:
I am verifying the validity or reason this question. Please wait few hours or 1day.

Reply:
1. strchr() have `NULL` return value.
2. `prm_get_string_value ()` do not have `NULL` return value.
3. `ha_node_list_pp` have `NULL` return value.

ha_node_list_p should be initialized `NULL` to compare strchr() return value.
ha_node_list_pp should be initialized `NULL` 

But ha_node_list_pp is not needed to be initialized `NULL`